### ISSUE TITLE
Add function and command line tool for converting and filtering EQASIM trips to STARLING user input file

### ISSUE THREAD

**Issue by vincent-leblond** [tellae/starling#95]
## Fonctionnalities:
- read eqasim trips file
- sample trips (seed and percent)
- filter using a polygon (from a geojson file)
- write starling geojson

## Recommandations:

- Read EQASIM trips file in geoparquet format for speed efficiency (already formated as linestrings)
```python
eqasim_flows = gpd.read_parquet(EQASIM_FILE)
```
- Need to convert to epsg 4326
```python
eqasim_flows = eqasim_flows.to_crs(4326)
```
- Columns must be added
```python
export["agent_type"] = "user"
export["icon"] = "user"
export["agent_id"] = ["u-" + str(i) for i in range(len(export))]
export["mode"] = "walk"
export["origin_time"] = export["departure_time"].astype(int)
```

**Comment by leo-desbureaux-tellae** [comment]
Some Eqasim trips are LineString of length > 2. In Starling we only manipulate Origin-Destination trips, ie Linestring length == 2.

Do we simplify the Eqasim trips to their outer ends ?

**Comment by leo-desbureaux-tellae** [comment]
Closed by #97
